completions/fish/meson.build

```diff
@@ -0,0 +1,4 @@
+fish_completion_dir = join_paths(get_option('datadir'), 'fish', 'completions')
```

Comment:
Perhaps this should be explicitly settable with an option, which could also be set to `no` to avoid installing fish completion — like how it's done for zsh completion.

Reply:
Added